rest/model/project.go

```diff
@@ -40,13 +40,20 @@ type ProjectTaskExecutionResp struct {
 }
 
 type APITriggerDefinition struct {
-	Project                      *string `json:"project"`
-	Level                        *string `json:"level"` //build or task
-	DefinitionID                 *string `json:"definition_id"`
-	BuildVariantRegex            *string `json:"variant_regex"`
-	TaskRegex                    *string `json:"task_regex"`
-	Status                       *string `json:"status"`
-	DateCutoff                   *int    `json:"date_cutoff"`
+	// project ID
```

Comment:
Are these supposed to be documented like the others (capitalization first, complete thought, period at the end)?